src/Nokia_LCD.cpp

```diff
@@ -112,6 +112,22 @@ void Nokia_LCD::begin() {
     sendCommand(0x0C);  // Set display control, normal mode.
 }
 
+void Nokia_LCD::couple(bool yes) {
+    if (yes && !coupled) {
+        digitalWrite(kCe_pin, LOW);
+    }
+    else if (!yes && coupled) {
+        digitalWrite(kCe_pin, HIGH);
+    }
+    coupled = yes;
```

Comment:
Hmm, couldn't this be changed to something simpler, i.e.:

```suggestion
    digitalWrite(kCe_pin, yes ? LOW : HIGH);
    coupled = yes;
```